grails-app/controllers/org/pih/warehouse/api/LoadDataApiController.groovy

```diff
@@ -54,7 +54,7 @@ class LoadDataApiController extends BaseDomainApiController {
         }
 
         if (Boolean.valueOf(config.productCatalogItems.enabled)) {
-            loadDataService.importProductCatalogItems(new URL(config.productSuppliers.url))
+            loadDataService.importProductCatalogItems(new URL(config.productCatalogItems.url))
```

Comment:
I'm confused. How were we encountering that error if we weren't even running the import on product catalog items?

Reply:
importProductCatalogItems was receiving invalid csv. It was not possible to extract productCode which lead to the error with saving product association (because product was null)

Reply:
Oh my. I didn't even think of that. Glad it wasn't any more complicated than that. :)